Linux/audit.sh

```diff
@@ -79,10 +79,39 @@ harden() {
 }
 
 
+backup_config_dirs() {
```

Comment:
I think maybe it might be worth it to make the function take in an arg as opposed to working on a global array.

Reply:
It is taking an arg, on line 86:
`local -n dir_arr=$1`